src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -113,7 +112,9 @@ public void setupSchema(List<ColumnMetadata> columns) {
                 column.getCollation(),
                 column.getOrdinal(),
                 null /* fieldId */,
-                parquetType.isPrimitive() ? parquetType.asPrimitiveType() : null));
+                parquetType.isPrimitive() ? parquetType.asPrimitiveType() : null,
+                clientBufferParameters.isEnableDistinctValuesCount(),
+                false /* enableValuesCount */));
```

Comment:
why not use clientBufferParameters.isEnableValuesCount() here ?

Reply:
Changed to `clientBufferParameters.isEnableValuesCount()`